Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "MIT OR Apache-2.0"
 readme = "README.md"
 
 [dependencies]
+log = "0.4"
```

Comment:
We used until now "eprintln" to report soft errors. Please either use "eprintln" also, or make log optional and then use the feature lag to either use log or eprintln consistently.

Reply:
Okey. I suppose as a general rule I'd rather have a library only ever output via logging APIs since I consider stdin/out to be owned by the application. If I were to add a feature I suppose that would then also require creating a logging shim for other places where eprintln is used. I can just use `eprintln` for updating this PR and then maybe a separate follow up to either switch to using `log` or adding `log` as a feature.